libs/blocks/table/table.css

```diff
@@ -17,7 +18,7 @@
 
 .table > .row {
   display: grid;
-  grid-template-columns: repeat(auto-fit, 50%);
+  grid-template-columns: repeat(auto-fit, minmax(50%, 1fr));
```

Comment:
not exactly specific to this PR, but I noticed that on the before / after page there's some horizontal scroll on mobile devices (iphone xr simulator - 414px width). We can track the issue separately though

Reply:
Yeah, this is because of https://github.com/adobecom/milo/blob/1cd1db9546000a70abf5f5c4b7658efae147719e/libs/blocks/table/table.css#L632-L633, `100%` + `4px` is more than the viewport width 😅 I'll check whether removing the lateral margin has any negative effects.

Reply:
This is a good catch, though, I've addressed it in a new commit